README.md

```diff
@@ -27,3 +27,25 @@ doi = {10.21105/joss.00456},
 URL = {http://www.theoj.org/joss-papers/joss.00456/10.21105.joss.00456.pdf}
 }
 ```
+
+
+## Code Formatting
+
+This repository uses `pre-commit` tool for code formatting.
+The tool checks formatting each time you commit to a repository.
+To install it locally use:
+  ```
+  pip3 install pre-commit  # (prepend `sudo` if you want to install it system wide)
+  ```
+
+To run it initially over the whole repo you can use:
+  ```
+  pre-commit run -a
+  ```
+
+If you get error that something is missing on your computer, do the following for:
+
+  - `clang-format-10`
+     ```
+     sudo apt install clang-format-10
```

Comment:
We use clang-format-14
```suggestion
  - `clang-format-14`
     ```
     sudo apt install clang-format-14
```

See here: https://github.com/ros-controls/control_toolbox/blob/ros2-master/.github/workflows/ci-format.yml

Reply:
Please let the reviewer close/mark the comments as resolved. You can just give a short remark like "done" or thumbs up emoji if you just agree. Sometimes is confusing when something is marked as done, but actually one doesn't sees it in the changes.

Reply:
Oh ok, this is how it works. I did not use it this way in the past.